Documentation/Storage-Configuration/ceph-teardown.md

```diff
@@ -108,8 +108,12 @@ DISK="/dev/sdX"
 # Zap the disk to a fresh, usable state (zap-all is important, b/c MBR has to be clean)
 sgdisk --zap-all $DISK
 
-# Wipe a large portion of the beginning of the disk to remove more LVM metadata that may be present
-dd if=/dev/zero of="$DISK" bs=1M count=100 oflag=direct,dsync
+# Wipe portions of the disk to remove more LVM metadata that may be present
+dd if=/dev/zero of="$DISK" bs=1K count=200 oflag=direct,dsync seek=0 # Clear at offset 0
+dd if=/dev/zero of="$DISK" bs=1K count=200 oflag=direct,dsync seek=$((1 * 1024**2)) # Clear at offset 1GB
+dd if=/dev/zero of="$DISK" bs=1K count=200 oflag=direct,dsync seek=$((10 * 1024**2)) # Clear at offset 10GB
+dd if=/dev/zero of="$DISK" bs=1K count=200 oflag=direct,dsync seek=$((100 * 1024**2)) # Clear at offset 100GB
+dd if=/dev/zero of="$DISK" bs=1K count=200 oflag=direct,dsync seek=$((1000 * 1024**2)) # Clear at offset 1000GB
```

Comment:
10TB hard drives aren't out of the question. I might suggest adding one more line 

```
dd if=/dev/zero of="$DISK" bs=1K count=200 oflag=direct,dsync seek=$((10000 * 1024**2)) # Clear at offset 10000GB
```

Reply:
My understanding is that it is hardcoded to go only up to 1000GB, looking at the [Ceph change](https://github.com/ceph/ceph/pull/55374/files#diff-6f3b1a5dcf5313dc06f71ba4997ee28ec219ecb5633a728b916465c684f1d87dR139) linked in #15545:  (file `src/os/bluestore/BlueStore.cc`, newly added lines 136-145)